doc/source/timeseries.rst

```diff
@@ -2436,22 +2436,22 @@ a convert on an aware stamp.
 
 .. note::
 
-   Using the ``.values`` accessor on a ``Series``, returns an NumPy array of the data.
+   Using :meth:`Series.to_numpy` on a ``Series``, returns a NumPy array of the data.
    These values are converted to UTC, as NumPy does not currently support timezones (even though it is *printing* in the local timezone!).
```

Comment:
This might need to be updated depending on the discussion of the default return value (object array of timestamps vs UTC converted naive datetime64)

Could also use `np.asarray` to make the point here in case we go for object array